curvefs/src/tools/topology/curvefs_build_topology_tool.cpp

```diff
@@ -0,0 +1,714 @@
+/*
+ *  Copyright (c) 2021 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: curve
+ * Created Date: 2021-09-03
+ * Author: wanghai01
+ */
+
+#include "curvefs/src/tools/topology/curvefs_build_topology_tool.h"
+
+DECLARE_string(mds_addr);
+DECLARE_string(cluster_map);
+DECLARE_uint32(rpcTimeOutMs);
+DECLARE_string(confPath);
+DECLARE_string(op);
+
+using ::curve::common::SplitString;
+
+namespace curvefs {
+namespace mds {
+namespace topology {
+
+void UpdateFlagsFromConf(curve::common::Configuration* conf) {
```

Comment:
can this union with `CurvefsToolRpc::UpdateFlagsFromConf`?

Reply:
> can this union with `CurvefsToolRpc::UpdateFlagsFromConf`?

for less work, it inherited from curvefs::tools::CurvefsTool.